apps/video/test/marionette/play_multiple_types_test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var assert = require('assert'),
+    Video = require('./lib/video');
+
+marionette('play various types of videos', function() {
+
+  var app, client, selectors;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    }
+  });
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    selectors = Video.Selector;
+    app = new Video(client);
+  });
+
+  function verifyVideo(type, done) {
+    var title = 'VID_0001';
+    var filename = title + '.' + type;
+
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/' + filename
```

Comment:
the test_media/ directory is not really owned by anyone. I'd prefer it if you put all of the videos that these tests depend on inside the video app directory, so that they don't get deleted or renamed by someone who doesn't know what they are there for.  I suggest creating a new directory apps/video/test/videos/ and putting the new files in that.